doc: Doc fixes for the FEM docs (#19729)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/fem/fem_nrf21540_gpio.rst

         };
      };

   When you add the above node to the devicetree and build the application, FEM-related Kconfig options are automatically enabled and FEM support is enabled.

>    Additionally, you can also enable the following Kconfig options if needed, in the :file:`prj.conf` file in the application folder:

Reviewer: Most of these Kconfigs have numerical value to be set. These have values rather than enable/disable.
These Kconfigs are given their own descriptions so I would rather advice the customer which Kconfigs should be considered but not necessarily needed to be set.
Because the way of setting them differs depending if they are to be set for nrf5340 network core or nrf52/nrf54 application core, I would omit the mention of prj.conf.
```suggestion
   Additionally, you can consider setting the following Kconfig options:
```
